spec/find_spec.rb

```diff
@@ -1,15 +1,22 @@
 require 'fourflusher'
 
 describe Fourflusher::SimControl do
-  let(:simctl) { Fourflusher::SimControl.new }
```

Comment:
Can we undo the changes in this file? This is an integration spec, and it should be mostly unaffected by your changes.